### ISSUE TITLE
What is mouse magnetize? Can't find any definition...

### ISSUE THREAD

**Issue by DRNadler** [GitHubLionel/wxMathPlot#37]
![image](https://github.com/user-attachments/assets/7757c914-f15a-4076-871b-5becbbaa2c17)

**Comment by GitHubLionel** [comment]
Just check the Magnetize checkbox in the configuration to see the effect.

**Comment by DRNadler** [comment]
Is the _only_ effect providing the cross-hairs? Did I miss something else?

**Comment by GitHubLionel** [comment]
Yes

**Comment by DRNadler** [comment]
Lionel, did we improve the comments and documentation so the next person will see what it is?
Thanks!

**Comment by GitHubLionel** [comment]
No. I think that the code is self explanatory and just run it to see what append.
